draft-irtf-cfrg-frost.md

```diff
@@ -977,8 +977,9 @@ Signature verification is as specified in {{prime-order-verify}}.
 ## FROST(Ed448, SHAKE256)
 
 This ciphersuite uses edwards448 for the Group and SHAKE256 for the Hash function `H`
-meant to produce Ed448-compliant signatures as specified in {{!RFC8032}}.
-The value of the contextString parameter is "FROST-ED448-SHAKE256-v11".
+meant to produce Ed448-compliant signatures as specified in {{!RFC8032}}. Note that this
+ciphersuite does not allow applications to specify a context string as is allowed for Ed448
+in {{RFC8032}}. The value of the contextString parameter is "FROST-ED448-SHAKE256-v11".
```

Comment:
This can be a bit confusing, because it's talking about two different context strings in sequence. Here's a suggestion but it's not great

```suggestion
meant to produce Ed448-compliant signatures as specified in {{!RFC8032}}. Note that this
ciphersuite does not allow applications to specify a context string as is allowed for Ed448
in {{RFC8032}}, and always sets the {{RFC8032}} context string to the empty string. The value of the (internal to FROST) contextString parameter is "FROST-ED448-SHAKE256-v11".
```